packages/client-core/src/common/components/Avatar/Avatar2.tsx

```diff
@@ -78,7 +78,7 @@ const Avatar = ({
         <div
           className={twMerge(
             'relative overflow-hidden rounded-2xl bg-gradient-to-b from-blue-400 to-blue-600',
-            size ? `w-[${size}px] h-[${size}px]` : 'h-32 w-32',
+            size ? `w-[${size}px] h-[${size}px]` : '',
```

Comment:
Dynamic classes like this dont actually get generated by Tailwind like you would expect here. You'll have to use styles to set the width and height.

Also, ideally we use `rem` over `px` where we can.

```
style={{ width: `${size}rem`, height: `${size}rem` }}